backend/lpd.c

```diff
@@ -1043,16 +1043,16 @@ lpd_queue(const char      *hostname,	/* I - Host to connect to */
       * Send the control file...
       */
 
-      if (lpd_command(fd, "\002%d cfA%03d%.15s\n", (int)strlen(control),
+      if (lpd_command(fd, "\002%lu cfA%03d%.15s\n", (unsigned long)strlen(control),
```

Comment:
The control file will NEVER need the range of unsigned long. Fine making this `%u` with an unsigned cast.